library/Ivoz/Cgr/Infrastructure/Persistence/Doctrine/Mapping/TpCdr.TpCdrAbstract.orm.yml

```diff
@@ -13,6 +13,9 @@ Ivoz\Cgr\Domain\Model\TpCdr\TpCdrAbstract:
     tpCdr_cgrid_idx:
       columns:
         - cgrid
+    tpCdr_createdAt_idx:
+      columns:
+      - created_at
```

Comment:
I think tp_cdrs.answer_time would be a better option (it is equal to kam_trunks_cdrs.start_time and it does not change whenever call is billed).

Reply:
Changed :)